client/idrepo/console/src/main/java/org/apache/syncope/client/console/panels/Realm.java

```diff
@@ -327,5 +337,27 @@ public boolean isVisible() {
             return SyncopeWebApplication.get().getSecuritySettings().getAuthorizationStrategy().
                     isActionAuthorized(this, RENDER);
         }
+
+        private boolean securityCheck(final String entitlement) {
+            return securityCheck(Set.of(entitlement));
+        }
+
+        private boolean securityCheck(final Set<String> entitlements) {
```

Comment:
It is not acceptable to check each single role via REST, you have all required information in session